extide/gradle/src/org/netbeans/modules/gradle/GradleBrokenRuntimeProblemProvider.java

```diff
@@ -54,10 +56,10 @@ public GradleJavaProjectProblemProvider(Project project) {
     @NbBundle.Messages({
         "LBL_BrokenPlatform=Broken Platform.",
     })
-    public Collection<? extends ProjectProblem> getProblems() {
-        List<ProjectProblem> ret = new ArrayList<>();
-        if (JavaRunUtils.getActivePlatform(project) == null) {
-            ret.add(ProjectProblem.createWarning(Bundle.LBL_BrokenPlatform(), Bundle.LBL_BrokenPlatform()));
+    public Collection<? extends ProjectProblemsProvider.ProjectProblem> getProblems() {
+        List<ProjectProblemsProvider.ProjectProblem> ret = new ArrayList<>();
```

Comment:
if this line is moved into if block, Collections.emptyList() could be returned in else block to avoid unnecessary allocation in case the default runtime is not broken

Reply:
There is no `JavaRuntimeManagerImpl` in the final PR. That was just good enough to get the things goin. Later on it was replaced by `JavaRuntimePlatformBridge`

Reply:
True. Code changed.